utils.go

```diff
@@ -206,11 +206,10 @@ func getRootlessStorageOpts(rootlessUID int, systemOpts StoreOptions) (StoreOpti
 		return opts, err
 	}
 	opts.RunRoot = rootlessRuntime
-	opts.GraphRoot = filepath.Join(dataDir, "containers", "storage")
 	if systemOpts.RootlessStoragePath != "" {
```

Comment:
only one question.  Should we honor the setting only when running as rootless?

Reply:
I'm sorry, but I don't understand your concern; the above piece of code is not removed, but simply moved to the `else` statement below.

Reply:
what I meant is that `systemOpts.RootlessStoragePath` is taken into account either if we are running as root or as rootless user, while I think it should be ignored when running as root.

Should we have instead something like `if isRootless && systemOpts.RootlessStoragePath != "" {` ?